### DISCUSSION TITLE
Rule - Can we copy a field content to another ?

### DISCUSSION THREAD

**Discussion by JeanMerand** [firefly-iii/firefly-iii#6672] [Firefly III] [answered]
Hi,

For most of my operations the Destination Account is good (imported via CSV) except when it is a check. 
In that cas this is is the Check Number.

I would like to create a rule telling that if "Destination Account Name" contains "Cheque n°" then append this text to "Notes". 
But I didn't find a way to refer to another column value.
Is it possible or not ?

Thank you.

![Screenshot 2022-11-24 103948](https://user-images.githubusercontent.com/85391288/203750103-e629873e-ea8e-413e-93e0-3c3f09405cf0.jpg)

**Comment by JC5** [comment, marked as answer]
https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/#more-complicated-rules-like-and-or-or-copypasting-values-etc

**Comment by JeanMerand** [comment]
Ok, so not possible because you want to keep it simple, I understand it.

Then I suggest to keep it simple but to add a 2nd choice in the "Action Value" of the "Rule will":
enter an expression (like it is today) or choose a field to copy over current value (and then you have a drop-list like "Action" or "Trigger" ones

something like this:


![Screenshot 2022-11-24 133721](https://user-images.githubusercontent.com/85391288/203786397-90e83689-213d-47ff-8ad6-8c42c42fa081.jpg)
